projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Boost_Cache_Actions.php

```diff
@@ -36,40 +37,40 @@
  *
  * @param int $post_id - The ID of the post to delete the cache for.
  */
-add_action( 'jetpack_boost_clear_page_cache_post', 'jetpack_boost_delete_cache_by_post_id' );
+add_action( 'jetpack_boost_clear_page_cache_post', 'jetpack_boost_invalidate_cache_by_post_id' );
 
 /**
  * Delete all cache files.
  */
 function jetpack_boost_delete_cache() {
 	$boost_cache = new Boost_Cache();
-	$boost_cache->delete_cache();
+	$boost_cache->invalidate_cache( Filesystem_Utils::DELETE_ALL );
 }
 
 /**
  * Delete cache for homepage and paged archives.
  */
-function jetpack_boost_delete_cache_for_home() {
+function jetpack_boost_invalidate_cache_for_home() {
 	$boost_cache = new Boost_Cache();
-	$boost_cache->delete_cache_for_front_page();
+	$boost_cache->invalidate_cache_for_front_page( Filesystem_Utils::DELETE_ALL );
 }
 
 /**
  * Delete cache for a specific URL.
  *
  * @param string $url - The URL to delete the cache for.
  */
-function jetpack_boost_delete_cache_for_url( $url ) {
+function jetpack_boost_invalidate_cache_for_url( $url ) {
 	$boost_cache = new Boost_Cache();
-	$boost_cache->delete_cache_for_url( $url );
+	$boost_cache->invalidate_cache_for_url( $url, Filesystem_Utils::DELETE_ALL );
 }
 
 /**
  * Delete cache for a specific post.
  *
  * @param int $post_id - The ID of the post to delete the cache for.
  */
-function jetpack_boost_delete_cache_by_post_id( $post_id ) {
+function jetpack_boost_invalidate_cache_by_post_id( $post_id ) {
 	$boost_cache = new Boost_Cache();
-	$boost_cache->delete_cache_by_post_id( (int) $post_id );
+	$boost_cache->invalidate_cache_by_post_id( (int) $post_id, Filesystem_Utils::DELETE_ALL );
 }
```

Comment:
Since these are only deleting, I think keeping the old names implying that they are meant to delete is even clearer. The same goes for the action hooks.

Reply:
Yeah, you're right! I'll fix that.